app/controllers/internal/path_redirects_controller.rb

```diff
@@ -0,0 +1,66 @@
+class Internal::PathRedirectsController < Internal::ApplicationController
+  layout "internal"
+
+  before_action :set_path_redirect, only: %i[edit update destroy]
+
+  after_action only: %i[update destroy create] do
+    Audit::Logger.log(:moderator, current_user, params.dup)
```

Comment:
I've never seen this before! What does `Audit::Logger` do here? 👀

Reply:
I hadn't either - it's also my first time! We are recording moderator/admin actions to the DB so we can keep track of changes - https://github.com/thepracticaldev/dev.to/pull/6797.

So here it's recording anytime an admin creates, updates, or destroys a `PageRedirect`. When debugging, we'll be able to see the admin action for a given redirect and which admin made the changes.

<img width="1173" alt="Screen Shot 2020-06-01 at 4 46 19 PM" src="https://user-images.githubusercontent.com/15987080/83453073-d4420e00-a427-11ea-815e-dfccfeebf93d.png">

Reply:
On second thought...thanks for drawing attention to this! I should change this from `:moderator` to `:internal` - this not an action that can be performed by moderators. Updated!